worlds/wargroove2/docs/wargroove2_en.md

```diff
@@ -0,0 +1,92 @@
+# Wargroove 2 Setup Guide
+
+## Required Files
+
+- Wargroove 2 installed through Steam on Windows
+  - Only the Steam Windows version is supported. MAC and Switch are not supported.
+- [The most recent Archipelago release](https://github.com/ArchipelagoMW/Archipelago/releases)
+
+## Backup playerProgress files
+`playerProgress` and `playerProgress.bak` contain save data for all of your Wargroove 2 campaigns. 
+Backing up these files is strongly recommended in case they become corrupted.
+1. Type `%appdata%\Chucklefish\Wargroove2\save` in the file browser and hit enter.
+2. Copy the `playerProgress` and `playerProgress.bak` files and paste them into a backup directory.
+
+## Update host.yaml to include the Wargroove 2 root directory
```

Comment:
It's not necessary. I added this in since Wargroove mods are a bit finicky with how they handle campaigns. I must've been confusing a mod issue with a save issue.

Reply:
I really hate the way GitHub shows a comment on a single line. This was meant to be commenting on the entire "Update host.yaml" section

Reply:
To be clear here, I thought the backup instruction was fine and honestly helpful. Sorry I forgot how GitHub displays a single-line comment